optuna/integration/pytorch_lightning.py

```diff
@@ -56,3 +59,87 @@ def on_validation_end(self, trainer: Trainer, pl_module: LightningModule) -> Non
         if self._trial.should_prune():
             message = "Trial was pruned at epoch {}.".format(epoch)
             raise optuna.TrialPruned(message)
+
+
```

Comment:
```suggestion

@experimental("2.10.0")
```

I think it would be better to start as an experimental feature.